File: Ruby_/entradaSaida/Lista7.rb
# frozen_string_literal: true

salarioMinimo = 1518
print "Digite seu salário: R$"
salario = gets.to_f

qtdSalMin = salario / salarioMinimo

puts "Com um salário de R$#{salario}"
puts "Você ganha %.2f salários mínimos" %qtdSalMin

File: Kotlin/entradaSaida/Lista7.kt
package entradaSaida
import java.net.URL

fun main(){
    val salariominimoatual = Regex(""""valor":"(\d+\.\d+)"""")
        .find(URL("https://api.bcb.gov.br/dados/serie/bcdata.sgs.1619/dados/ultimos/1?formato=json").readText())
        ?.groupValues?.get(1)?.toDouble() ?: 1412.00
    print("Digite seu salário: R$")
    val salario = readln().toDouble()
    val qtdSalMin = salario / salariominimoatual

    println("Com um salário de R$$salario,\nVocê ganha %.2f salários mínimos".format(qtdSalMin))
}